lib/takeaway.rb

```diff
@@ -0,0 +1,65 @@
+class Takeaway
+  
+  attr_reader :menu, :basket
+  DEFAULT = 1
+
+  def initialize
+    @menu = { 'Fish' => 1, 
+             'Burger' => 2, 
+             'Chips' => 3, 
+             'Pizza' => 8, 
+             'Pasta' => 0.5
+            }
+    @basket = []
+  end
+
+  def display_menu
+    @menu
+  end
+
+  def order(item, quantity = DEFAULT)
+    @item = item
```

Comment:
@item, This appears to be an class instance variable - use of @, it is however not initialised in the initialise method